docs/AccountStatus.md

```diff
@@ -0,0 +1,15 @@
+# Freeclimb::AccountStatus
+
+## Properties
+
+| Name | Type | Description | Notes |
+| ---- | ---- | ----------- | ----- |
```

Comment:
There is still not documentation with the enum values.

Reply:
Is this good for the enum documentation structure:

# Freeclimb::AccountStatus

## Properties

| Name | Type | Description | Notes |
| ---- | ---- | ----------- | ----- |
    **CLOSED** | **AccountStatus** | The status of this account. It is one of: active, suspended, or closed. | "closed" |
    **SUSPENDED** | **AccountStatus** | The status of this account. It is one of: active, suspended, or closed. | "suspended" |
    **ACTIVE** | **AccountStatus** | The status of this account. It is one of: active, suspended, or closed. | "active" |

## Example

```ruby
require 'freeclimb'

instance = Freeclimb::AccountStatus.new()
```

Reply:
thats a great format. lets apply that every where

Reply:
Updated in latest commit